contracts/MyStrategy.sol

```diff
@@ -453,18 +500,43 @@ contract MyStrategy is BaseStrategy, ReentrancyGuardUpgradeable {
         emit RewardsCollected(token, amount);
     }
 
-    /// @dev Send the BADGER token to the badgerTree
-    function _sendBadgerToTree(uint256 amount) internal {
-        // Transfer to tree without taking any fee
-        IERC20Upgradeable(BADGER).safeTransfer(IVault(vault).badgerTree(), amount);
-        emit TreeDistribution(BADGER, amount, block.number, block.timestamp);
+    /// @dev Takes a fee on the token and sends remaining to the given briber recepient
+    function _sendTokenToBriber(address token, address recepient, uint256 amount) internal {
+        // Process redirection fee
+        uint256 redirectionFee = amount.mul(redirectionFees[token]).div(MAX_BPS);
+        if (redirectionFee > 0) {
+            IERC20Upgradeable(token).safeTransfer(IVault(vault).treasury(), redirectionFee);
+            emit RedirectionFee(
+                IVault(vault).treasury(),
+                token,
+                redirectionFee,
+                block.number,
+                block.timestamp
+            );
+        }
+
+        // Send remaining to bribe recepient
+        // NOTE: Calculating instead of checking balance since there could have been an 
+        // existing balance on the contract beforehand (Could be 0 if fee == MAX_BPS)
+        uint256 redirectionAmount = amount.sub(redirectionFee);
+        if (redirectionAmount > 0) {
+            IERC20Upgradeable(token).safeTransfer(recepient, redirectionAmount);
+            emit TokenRedirection(
+                recepient,
+                token,
+                redirectionAmount,
+                block.number,
+                block.timestamp
+            );
+        }
     }
 
     function _sweepRewardToken(address token) internal {
         _onlyNotProtectedTokens(token);
 
         uint256 toSend = IERC20Upgradeable(token).balanceOf(address(this));
-        _handleRewardTransfer(token, toSend);
+        address recepient = bribesRedirectionPaths[token];
+        _handleRewardTransfer(token, recepient, toSend);
```

Comment:
This function doesn't notify the bribes processor on purpose (see [comments](https://github.com/Badger-Finance/vested-aura/blob/f8152243d0627e1945b0a6926e2564381caa7faf/contracts/MyStrategy.sol#L167)). The reason is that it can be called outside of a bribes round. For instance, on bveCVX we use it sweep the cvxFXS (received as an additional reward) into the processor to then handle it alongside with the bribes. We do it atomically with the claiming of bribes but we may want to do it separately at some point, if there's no bribes for whatever reason, and then rageQuit the processor to handle the cvxFXS manually. So not notifying will allow us to do so.

Reply:
Makes sense!